packages/sysweb3-utils/src/getContract.ts

```diff
@@ -0,0 +1,51 @@
+import Web3 from 'web3';
+
+import { getErc21Abi, getErc55Abi } from './contracts';
+
+export const getContract = async (
+  contractAddress: string,
+  networkUrl: string
+) => {
+  const provider = new Web3.providers.HttpProvider(networkUrl);
+
+  const web3Provider = new Web3();
+
+  web3Provider.setProvider(provider);
+
+  const [abi721, abi1155] = [getErc21Abi(), getErc55Abi()];
+
+  const abisInfos = [
+    { abi: abi721, interfaceId: '0x80ac58cd', type: 'ERC-721' },
```

Comment:
Additionally to the base extension erc721 extensions interface as well:
URIStorage: 0xc87b56dd
Enumerable: 0x79f154c4
Burnable: 0x42966c68
Royalty: 0x01ffc9a7

Reply:
Fixed